tests/integration/test_storage_s3/test.py

```diff
@@ -2117,10 +2117,12 @@ def test_read_subcolumns(started_cluster):
     assert res == "0\troot/test_subcolumns.jsonl\t(0,0)\ttest_subcolumns.jsonl\t0\n"
 
     res = instance.query(
-        f"select x.b.d, _path, x.b, _file, x.e from s3('http://{started_cluster.minio_host}:{started_cluster.minio_port}/{bucket}/test_subcolumns.jsonl', auto, 'x Tuple(b Tuple(c UInt32, d UInt32), e UInt32) default ((42, 42), 42)')"
+        f"select x.b.d, _path, x.b, _file, dateDiff('minute', _time, now()), x.e from s3('http://{started_cluster.minio_host}:{started_cluster.minio_port}/{bucket}/test_subcolumns.jsonl', auto, 'x Tuple(b Tuple(c UInt32, d UInt32), e UInt32) default ((42, 42), 42)')"
```

Comment:
This test is flaky: https://s3.amazonaws.com/clickhouse-test-reports/0/9a55cdf77cc9452df8e1c61dca7864bb035357c3/integration_tests__asan__%5B3_4%5D.html
We cannot rely on the time here, difference between the time the file was created and this query execution can be anything.
And how is reading `_time` virtual column is related to reading subcolumns (test `test_read_subcolumns` tests reading subcolumns, not virtual columns)? Please, use a separate test function for it and make it less dependent on query execution time.